.github/workflows/notify_discord.yml

```
@@ -0,0 +1,102 @@
+            1.
+            2.
+            3.
+
+            Texto da PR:
+            ${{ github.event.pull_request.body }}
+
+      # 🛟 Fallback (caso IA falhe)
+      - name: Definir mensagem final
+        id: final-message
+        if: github.event.action == 'opened'
+        run: |
```

actionlint (1.7.12)

[error] property "text" is not defined in object type {response: string; response-file: string}

(expression)